packages/dcos-sdk/README.md

```diff
@@ -0,0 +1,52 @@
+# DC/OS UI SDK
+
+DC/OS plugin developer's best friend!
+
+## NavigationService
+
+ExtensionProvider that provides extension point for navigation.
+
+Most probably you don't want to use it directly but rather implement its `INavigationExtension` and bind it to `ExtensionProvider` named `NavigationExtensionProvider`.
+
+### Example NavigationExtension
+
+```ts
+class PluginNavigation implements INavigationExtension {
+  getElements() {
+    return [
+      {
+        path: "/plugin-path",
+        link: "Plugin name for the Sidebar",
+        parent: "root",
+        icon: <Icon />
+      },
+      {
+        parent: "/plugin-path",
+        path: "/plugin-path/sub-page",
+        link: "Plugin page"
+      }
+    ];
+  }
+}
+```
+
+## RoutingService
```

Comment:
From reading this doc it’s it clear what the difference between an extension provider and a service is if there is one. Could you please adjust the document so that’s clear?